lib/Slice.ark

```diff
@@ -0,0 +1,15 @@
+{
+    #made by Natendrtfm (on Github) with <3
+    (let slice (fun (inputed length) {
```

Comment:
same here, rename it `sliceStr`, would be better imho because it only works with strings here

Reply:
also, you're slicing from 0, we should be able to slice from a random index, like `(slice my-string 1 5)`

Reply:
Done both
i haven't thinked about slicing from a random index when i make it, thank for your feedback !